Song.js

```diff
@@ -29,6 +30,18 @@ class Song
 			this.makePeriscopeStream(callback);
 		} else if (url.endsWith(".mp3") && (url.startsWirth("http://") || url.startsWith("https://"))) {
 			this.makeMP3Stream(callback);
+		} else if (config.youtube && config.youtube.token) {
+			console.log("Searching in YouTube: " + url);
+			var options = {
+				maxResults: 1,
+				key: config.youtube.token
+			};
+			yt_search(url, options, (error, results) => {
+				if (error) return console.log(error);
+				if (results.length <= 0) return console.log('No YouTube results for ' + url);
```

Comment:
Please be consistent in formatting; use `{ }` and `"` instead of `'`. Also, why return the return value of console.log? This would be better:
```js
if (error) {
  console.log("Failed to search Youtube: \"" + error + "\"");
  return;
}
```

Reply:
Ok